packages/shared/src/contexts/AuthContext.tsx

```diff
@@ -120,7 +122,8 @@ export const AuthContextProvider = ({
 }: AuthContextProviderProps): ReactElement => {
   const [loginState, setLoginState] = useState<LoginState | null>(null);
   const endUser = user && 'providers' in user ? user : null;
-  const referral = user?.referrer;
+  const referral = user?.referralId || user?.referrer;
```

Comment:
From what I can see `user?.referrer` is  no longer used, if we can simplify this it would be great. There is a comment on API side about this also.

Reply:
If you confirmed that it is not used then you can simplify :)

Reply:
Gonna add it to backlog to investigate and confirm properly https://dailydotdev.atlassian.net/browse/WT-1431